Calculator.rb

```diff
@@ -0,0 +1,70 @@
+# Calculator Exercise
+def is_operator(list,user_input)
```

Comment:
The variable name `list` is a little confusing at this point because the method expects a hash, not an array/list